it/src/main/java/org/corfudb/universe/node/server/process/ExecutionHelper.java

```diff
@@ -0,0 +1,59 @@
+package org.corfudb.universe.node.server.process;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.tools.ant.Project;
+import org.apache.tools.ant.taskdefs.Copy;
+import org.apache.tools.ant.taskdefs.Execute;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Optional;
+
+
+/**
+ * Provides the helper functions that do operations (copy file/execute command) on a remote machine.
+ */
+@Slf4j
+public class ExecutionHelper {
+    private static final Project PROJECT = new Project();
+    private static final ExecutionHelper INSTANCE = new ExecutionHelper();
+
+    private ExecutionHelper() {
+        //prevent creating class instances
+    }
+
+    public static ExecutionHelper getInstance() {
+        return INSTANCE;
+    }
+
+    /**
+     * Copy a file
```

Comment:
More documentation. Is this a local file copy?

Reply:
done.
it's on a local machine (added that info into ExecutionHelper javadoc).